docs/spec/dns-integration.md

```diff
@@ -0,0 +1,79 @@
+DNS Integration for NSM
+============================
+
+Specification
+-------------
+
+Network Service Mesh needs to be able to provide a workload with DNS service from Network Services Without breaking K8s DNS
+
+Implementation details (optional)
+---------------------------------
+
+####nsm-coredns
```

Comment:
Seems we need a space char here, it isn't formatted properly now

Reply:
thanks, fixed